examples/nullable.md

```diff
@@ -0,0 +1,45 @@
+# Nullable type attribute
+
+The `nullable` type attribute can be used to outline the instance of this type *Value* MAY be unset (e.g. `null` or `nil`).
+
+## Unset a `string` or an `array` type
+
+### MSON
+
+```mson
+- keyA (string, nullable, required) - Value of this field can be either unset or of the `string` type
```

Comment:
I tend to lean more towards the usage of 'null' and 'nil' instead of 'unset', so I'm happy to change that if everyone thinks it is best, however it was also mentioned we are not actually introducing a new type, so using those terms could be confusing. Perhaps their is just no clear way to explain it.